.gitignore

```diff
@@ -24,6 +24,7 @@
 
 # Ignore storage (uploaded files in development and any SQLite databases).
 /storage/*
+!/storage/te/st/*
```

Comment:
While writing tests for anthropic I extended our fixtures to include ActiveStorage attachments for our conversation fixtures which include attachments. These ActiveStorage attachments point to actual files in the filesystem. They're all in the `te/st` directories so that's why this is allowed.